src/storage/page_buffer.c

```diff
@@ -14154,8 +14154,11 @@ pgbuf_rv_dealloc_undo_compensate (THREAD_ENTRY * thread_p, LOG_RCV * rcv)
  */
 int
 pgbuf_fix_if_not_deallocated_with_caller (THREAD_ENTRY * thread_p, const VPID * vpid, PGBUF_LATCH_MODE latch_mode,
-					  PGBUF_LATCH_CONDITION latch_condition, PAGE_PTR * page,
-					  const char *caller_file, int caller_line)
+					  PGBUF_LATCH_CONDITION latch_condition, PAGE_PTR * page
+#if !defined (NDEBUG)
+					  , const char *caller_file, int caller_line
+#endif
+  )
```

Comment:
If caller_file is added here, Instead of adding `btree_` prefix, you can utilize the gdb conditional breakpoint and convineient functions for debugging.

For example:
Use a break condtiion with $_streq (please refer to the https://sourceware.org/gdb/onlinedocs/gdb/Convenience-Funs.html)
```
b pgbuf_fix_if_not_deallocated_with_caller if $_streq(caller_file, "btree.c")
b pgbuf_fix_if_not_deallocated_with_caller if $_streq(caller_file, "btree_load.c")
```

If you would like to do something (such as write some logs..) when that breakpoint is reached, please write a gdb script. You can refer to the contrib/gdb_scripts.